public/video-ui/src/components/utils/GridImageSelect.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Modal from './Modal';
+
+const GRID_URL = 'https://media.gutools.co.uk';
```

Comment:
There's a conf file injected into the head which already contains the reauth URL, this could be added there.